pubtools/pulplib/_impl/model/repository/yum.py

```diff
@@ -70,6 +70,18 @@ class YumRepository(Repository):
 
     type = pulp_attrib(default="rpm-repo", type=str, pulp_field="notes._repo-type")
 
+    population_sources = pulp_attrib(
+        default=(), type=tuple, converter=tuple, pulp_field="notes.population_sources"
```

Comment:
Can you please make this consistent with the other "immutable collection" attributes, i.e. use FrozenList?

So that would be something like:

```suggestion
        default=attr.Factory(FrozenList), type=list, converter=FrozenList, pulp_field="notes.population_sources"
```